ui/src/app/components/end-migration/end-migration.component.html

```diff
@@ -0,0 +1,25 @@
+<h2>End Migration</h2>
+<div>
+    <b>Source database:</b>  {{sourceAndTargetDetails.SourceDatabaseName}}({{sourceAndTargetDetails.SourceDatabaseType}})
+    <br>
+    <b>Spanner database:</b>  <a [href]="sourceAndTargetDetails.SpannerDatabaseUrl">{{sourceAndTargetDetails.SpannerDatabaseName}}</a>
+</div>
+<br>
+<div>
+    <b>Please follow these steps to complete the migration:</b>
+    <ol>
+        <li>Validate that the schema has been created on Spanner as per the configuration</li>
+        <li>Validate the data has been copied from the Source to Spanner. You can use the <a href="https://github.com/GoogleCloudPlatform/professional-services-data-validator">Data Validation Tool</a> to help with this
+            process.</li>
+        <li>Stop the writes to the source database. This will initiate a period of downtime.</li>
```

Comment:
Highlight that fact that the downtime will begin here

Reply:
done